### PR TITLE
Syntax-coloring Objective-C code

### PR DESCRIPTION
**Pull request by JaviSoto**
Just a tiny contribution to an otherwise AMAZING write-up. Really good realistic introduction to FP (or FRP) for Obj-C developers. The community should be thankful for this. I know I am :)

### CODE REVIEW

File: Postmodern Programming.md [hunk 1]
```diff
@@ -187,9 +202,11 @@ There’s an interesting contrast to be drawn between _simplicity_ and _ease_. I
 
 How many concrete concepts does `-firstObject` contain? When I say “concrete” I mean more or less “invariant”—concepts which are part of the contract of the method. Remember, this is, by definition, semantics, so we shouldn’t be afraid of this question just because it’s fuzzy. Let’s count them:
 
-    -(id)firstObject {
-    	return self.count? self[**0**] : **nil**;
-    }
+```objc
+-(id)firstObject {
+	return self.count? self[**0**] : **nil**;
```
Comment (robrix) [inline Postmodern Programming.md:207]: This seems to have broken the bolding of 0 and nil. Do you think that’s something that’s resolvable? If not, there’s other ways of highlighting those, either with something like clang’s ~~~ markers or a quick line describing them.

File: Postmodern Programming.md [hunk 2]
```diff
@@ -296,18 +323,22 @@ To the second question: If state is mutated in the forest and no one is around t
...
 In the meantime, we’ll still just return a dictionary from this method; let’s call it `-navigationItemState`:
 
-    -(NSDictionary *)navigationItemState {
-    	return @{ @"title": @"Recipient",
-    	         @"prompt": @"Where would you like the flowers sent?" };
-    }
+```objc
+-(NSDictionary *)navigationItemState {
+	return @{ @"title": @"Recipient",
+	         @"prompt": @"Where would you like the flowers sent?" };
```
Comment (robrix) [inline Postmodern Programming.md]: Sadly, the indentation seems to have gotten a little garbled by this. Not the end of the world, tho!

**Comment by robrix** [comment]
This is a great addition. Thank you very much for submitting it! I will wait to merge until you’ve responded about the 0/nil thing (don’t feel that you necessarily need to implement them, I’m just wondering if you have any ideas :) ); the indentation mentioned in my other comment is, by contrast, not something I consider blocking.

**Comment by robrix** [comment]
And thank you very much for the kind words, too. It means a lot to me to know it was well-received! :)

File: Postmodern Programming.md (hunk 1 above)
Comment (JaviSoto) [inline Postmodern Programming.md:207]: Yeah, I didn't notice this. Anything you use won't be used as formatting :-/

File: Postmodern Programming.md [hunk 3]
```diff
@@ -296,18 +323,22 @@ To the second question: If state is mutated in the forest and no one is around t
...
 In the meantime, we’ll still just return a dictionary from this method; let’s call it `-navigationItemState`:
 
-    -(NSDictionary *)navigationItemState {
-    	return @{ @"title": @"Recipient",
-    	         @"prompt": @"Where would you like the flowers sent?" };
-    }
+```objc
+-(NSDictionary *)navigationItemState {
+	return @{ @"title": @"Recipient",
+	          @"prompt": @"Where would you like the flowers sent?" };
```
Comment (JaviSoto) [inline Postmodern Programming.md:329]: Fixed :)

File: Postmodern Programming.md (hunk 1 above)
Comment (JaviSoto) [inline Postmodern Programming.md:207]: How would you like to fix this? IMO reading the *\* in the code is obvious to me that is just to emphasize, not part of the code itself fwiw.

Comment (robrix) [inline Postmodern Programming.md:207]: I think I’ll merge as-is, and then tweak a little myself. This is super kind of you; thanks!

Comment (JaviSoto) [inline Postmodern Programming.md:207]: Great :)

**Pull request merged**